### PR TITLE
Mark mvneta and mvpp2 as supported

### PR DESCRIPTION
**Pull request by teknoraver**

### CODE REVIEW

File: people.org [hunk 1]
```diff
@@ -83,9 +85,9 @@ People to keep in loop:
 
 Drivers that will get XDP and use page_pool:
  - driver: mlx5
```
Comment (tohojo) [inline people.org:87]: Is mlx5 really still in 'will get XDP and use page_pool' state? (ping @netoptimizer )

Could we update this along with the rest?

Comment (teknoraver) [reply people.org:87]: Any update? Can we update mlx5 in a different PR?
I got permission from Lorenzo to update his drivers, but never talked with anyone about mlx5.

Comment (netoptimizer) [reply people.org:87]: Sorry for late answer.
Sure we can update mlx5 in a different PR.

The file drivers.org seems very out of date :-(
Can you update it some more?

**Comment by netoptimizer** [comment]
I'm going to merge... and hope @teknoraver will update drivers.org in another PR

**Pull request merged**